src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -2,224 +2,265 @@ import React from 'react'
+                          display: grid;
+                          grid-gap: ${1.5 * GU}px;
+                          grid-template-columns: auto auto;
+                        `}
+                        >
+                          By <LocalIdentityBadge entity={author} />
+                        </h2>
+                      )}
+                    </div>
+                  </div>
+                  {canUpgrade && onRequestUpgrade && (
+                    <Button
+                      mode="strong"
+                      onClick={onRequestUpgrade}
+                      css={`
+                        width: 128px;
+                        margin: ${2.5 * GU}px 0;
+                        align-self: center;
+                      `}
+                    >
+                      Install App
+                    </Button>
+                  )}
                 </div>
-              </React.Fragment>
-            )}
-          </div>
-        </div>
-        <div
-          css={`
-            padding: ${compact ? `${3 * GU}px 0 0 ${80 + 3 * GU}px` : '0'};
-          `}
-        >
-          {canUpgrade && onRequestUpgrade && (
-            <Button mode="strong" onClick={onRequestUpgrade}>
-              Upgrade
-            </Button>
-          )}
-        </div>
-      </div>
-      {screenshots.length > 0 && (
-        <Screenshots repo={repo} screenshots={screenshots} />
-      )}
-      <div
-        css={`
-          display: flex;
-          flex-direction: column;
-          justify-content: space-between;
-          width: 100%;
-          flex-direction: ${compact ? 'column' : 'row'};
-        `}
-      >
-        <DetailsGroup compact={compact}>
-          <Heading2>Description</Heading2>
-          <div>{description}</div>
-
-          {!!repoDetails && (
-            <React.Fragment>
-              <Heading2>Details</Heading2>
-              <Markdown text={repoDetails} />
-            </React.Fragment>
-          )}
-        </DetailsGroup>
-        <DetailsGroup compact={compact}>
-          <Heading2>Installed instances</Heading2>
-          {instances.map(({ proxyAddress }) => (
-            <div
-              key={proxyAddress}
-              css={`
-                & + & {
-                  margin-top: ${2 * GU}px;
-                }
-              `}
-            >
-              <LocalIdentityBadge entity={proxyAddress} />
-            </div>
-          ))}
-
-          <Heading2>Source code</Heading2>
-          <div>
-            {sourceUrl ? (
-              <StyledLink href={sourceUrl}>{sourceUrl}</StyledLink>
-            ) : (
-              'No source code link.'
-            )}
-          </div>
+                {screenshots.length > 0 && (
+                  <Screenshots repo={repo} screenshots={screenshots} />
+                )}
+                <div
+                  css={`
+                    display: grid;
+                    grid-template-columns: ${compact ? 'auto' : '60% 40%'};
```

Comment:
```suggestion
                    grid-template-columns: ${compact ? 'auto' : '60% auto'};
```

I think this will let the right side be automatically sized for the remaining space, including the gap, so that we don't need to do the manual `width` overrides on `BreakLink` and the `IdentityBadge`.

Reply:
Good catch! and TIL